mycore-base/src/main/java/org/mycore/common/content/MCRContent.java

```diff
@@ -260,11 +260,20 @@ protected MCRDataURLEncoding getDataURLEncoding() throws IOException {
 
     /**
      * Parses content, assuming it is XML, and returns the parsed document.
-     * 
+     *
```

Comment:
I fail to see the mentioned API change. 

Actually, there is another API change, because some child classes had deviating lists of thrown exception classes. That's why there are changes to, say, `MCRMailer`.